test_sum_terms.py

```diff
@@ -0,0 +1,3 @@
+Test.assert_equals(series_sum(1), "1.00")
```

Comment:
You're using a `Test` object even though `Test` exists nowhere in your file. Did you try running these in your own console?